Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/tools/LineToolIntegrationTest.java

```diff
@@ -185,6 +197,8 @@ public void testCheckmarkLineFeatureChangeColor() {
 
 	@Test
 	public void testCheckmarkLineFeatureChangeShape() {
+		CountingIdlingResource idlingResource = launchActivityRule.getActivity().getIdlingResource();
+		IdlingRegistry.getInstance().register(idlingResource);
```

Comment:
You already invoked .register() in the setup method, so I think there is no need to additionally .register() at the start of the test.